patching.Rmd

```diff
@@ -1,7 +1,7 @@
 
 # Basic workflow
 
-Update the `trunk` branch to the latest sources with `git svn rebase`. Create a new branch for your changes and develop as you would normally with git commits.
+Update the `trunk` branch to the latest sources with `git rebase`. Create a new branch for your changes and develop as you would normally with git commits.
```

Comment:
```suggestion
Update the `trunk` branch to the latest sources with `git pull`. Create a new branch for your changes and develop as you would normally with git commits.
```

Reply:
Does this need to be `pull --rebase` or even `pull --ff-only` ? We don't want Git merges lingering around.

Reply:
Good point, we could add `--rebase` for safety. Development should be done from a branch but it's true we often forget to create a new branch when starting work.

Reply:
```suggestion
Update the `trunk` branch to the latest sources with `git pull --rebase`. Create a new branch for your changes and develop as you would normally with git commits.
```

Reply:
Somehow I can't commit the suggestions:

> You do not have permission to push to this repository.

Can you do it please? Otherwise I'll pull the branch locally.